app/views/lookup_keys/_value.html.erb

```diff
@@ -1,13 +1,42 @@
-<div class="fields">
-  <%= text_f f, :match, :size => "col-md-8",
-    :help_block => popover(_("Explain matchers"), _("Matcher is a combination of an attribute and its value, if they match, the value below would be provided.<br> You may use any attribute foreman knows about, such as facts etc for example: <code> domain = example.com </code> or <code> is_virtual = true</code>"),
-                           :title => _("Matcher"))
-  %>
-  <%= checkbox_f(f, :use_puppet_default, :label => _('Use Puppet default'), :size => "col-md-8",
-                    :help_block => use_puppet_default_help,
-                    :onchange=>'toggleUsePuppetDefaultValue(this, "value")') if is_param %>
-  <%= textarea_f f, :value, :rows => :auto, :value => f.object.value_before_type_cast, :size => "col-md-8",
-                    :disabled => f.object.use_puppet_default,
-                    :fullscreen => :true,
-                    :help_inline => remove_child_link(icon_text("remove"), f, {:title => _('remove value')}) %>
-</div>
+<% if is_template %>
+  <table>
+<% end %>
+    <tr class="fields matchers form-group <%= 'has-error' if f.object.errors.any? %>">
+      <td>
+        <%= f.hidden_field :match, :class => 'match' %>
+        <%= select_tag '', nil, :class => 'matcher_key' %>
```

Comment:
When rebased onto develop, this doesn't seem to be activating with select2:

![screenshot from 2015-06-25 12 14 18](https://cloud.githubusercontent.com/assets/482089/8353229/bf079abe-1b33-11e5-989f-7e65f58b020b.png)